Lib/asyncio/controller.py

```diff
@@ -0,0 +1,78 @@
+"""Protocol server controller."""
+
+__all__ = ['Controller']
+
+
+import asyncio
+import os
+import socket
+import threading
+
+try:
+    from socket import socketpair
+except ImportError:
+    from asyncio.windows_utils import socketpair
```

Comment:
Since it's only needed in Python <= 3.4, I'll remove it.  Even if it's needed in other places, I don't care about 3.4 any more.